### DISCUSSION TITLE
Report - (4th Sem.) Week 8📢

### DISCUSSION THREAD

**Discussion by giomskii** [SE-TINF22B6/DHBWhub#804] [Weekly Report]
Greetings and welcome to our fourteenth progress report! We're thrilled to update you on the latest news and developments since last week! 🥳

### Summary of the retrospective meeting 🎈 
In this section, you can find a summary of our retrospective meeting from the last lecture. You can also see the results in the picture below:  
  
![grafik](https://github.com/SE-TINF22B6/DHBWhub/assets/122597204/ccf41bb8-deba-4d0e-ab24-2dabaf2db1ad)
  
**What helped us move forward?**
- Our weekly meetings and discussions
- Constructive feedback (especially within Pull Requests)
- Coding sessions when trying to fix or find a way to solve a more complex issue/problem
- (ChatGPT)

**What held us back?**
- Minor setbacks because of technologies we aren't that familiar with
- Too much documentation
- Other subjects

**How could we do things differently?**
- More team work and precise/clear communication
- Invest more time learning the new technologies
- Frequent and smaller updates on main branch

**What should we do next?**
- Preparation of the final demo, handout and presentation
- Focus on finishing the key features

### Latest developments 📈 
In quick fashion, let me show you some of the things we have achieved thus far!

#### Frontend Frontend Frontend 🖥️
You may sign in via your google account to ensure more comfort!
  
![grafik](https://github.com/SE-TINF22B6/DHBWhub/assets/122597204/ffb76510-2549-441b-9e65-1a3246e4cb2a)
  
![grafik](https://github.com/SE-TINF22B6/DHBWhub/assets/122597204/a760e191-4c21-4624-a94e-d6a963dd81ef)

You are now finally able to create and update a post with a title, description and an optional image!

![grafik](https://github.com/SE-TINF22B6/DHBWhub/assets/122597204/0b869d0f-48b8-4833-9f97-d4e6e1cd03c2)
  
You can also view events in the thread view in which you are able to see the location on google street view!
  
![grafik](https://github.com/SE-TINF22B6/DHBWhub/assets/122597204/8e447513-86f6-4e22-a2fa-f822072eac2d)
  

We have implemented a profile page, in which you can see and change your own data as you can see in the image below:
  
![grafik](https://github.com/SE-TINF22B6/DHBWhub/assets/122597204/eff0ba98-72b1-420c-ac5f-a49bda8c77d4)
  
You can now report posts and comments! These will be send per mail to the mail adress of the DHBWhub admins. 
  
![grafik](https://github.com/SE-TINF22B6/DHBWhub/assets/122597204/75a816fa-2c33-4792-b3cc-fbcebeb4f543)
  
![grafik](https://github.com/SE-TINF22B6/DHBWhub/assets/122597204/ff35b922-0b47-4021-b1e9-8a8b33435117)
  

### Sprint 10: What have we achieved ❓ What's to come❔

✅ Frontend: Implement ads and ad blocker pop up
✅ Frontend: Initialize user page
✅ Frontend: Implement Calendar Event page
✅ Frontend: Fix CORS errors
✅ Frontend: Implement Profile page
✅ Frontend: Implement report function
✅ Frontend: Implement comment menu for reporting comments
✅ Frontend: Implement functionality to create a post
✅ Frontend: Initialize user page and search page
✅ Backend: Improve User Endpoints
✅ Backend: Delete User Endpoints
✅ Backend: Fix image Endpoints
✅ Backend: Sign in via Google
🚧 Frontend: Implement Search page
🚧 Frontend: Implement Tag-Overview page
🚧 Frontend: Implement notifications window and friends page
🚧 Frontend: Implement report function
🚧 Frontend: Implement Calendar and User page
🚧 Remove Herobrine

**Comment by mirak55** [comment]
Dear DHBWhub team,
you gave an good overview on your project review, and really moved on on your website itself. Its nice to see so many functional additions to your website and it looks like you probably can finish most of your remaining sprint tasks till the Demo. Keep up the good work and I am looking forward to see your Demo next week.
Your CookPit team

**Comment by BenjaminB109** [comment]
Dear DHBWHub Team,
You have given a good overview of the Retrospective. Glad to see that you have made good progress on the frontend.

**Comment by TadaXD20** [comment]
Dear DHBWHub team,
your retrospective board shows that you have really put a lot of thought into your development process. Well done! You have a very good overview what you achieved and what needs to be done before the next appointment. So I'm sure you're finishing on time :)
Best regards from the Plapy team
